conan/internal/cache/db/cache_database.py

```diff
@@ -91,6 +91,9 @@ def create_package(self, path, ref: PkgReference, build_id):
         self._packages.create(path, ref, build_id=build_id)
 
     def list_references(self, pattern=None):
+        """Returns a list of RecipeReference, optionally filtering by pattern.
+         The references are sorted by timestamp,
```

Comment:
```suggestion
         The references are sorted by version,
```
Really sorted by timestamp, but timestamp and version removed? Does this make sense?